### PR TITLE
os: add instructions to disable socket-activated sshd

### PR DESCRIPTION
**Pull request by kbrwn**
Adds an additional section to Customizing the SSH Daemon doc.

This provides a work-around for: https://github.com/coreos/bugs/issues/966 which was recently brought up on the mailing list. 

Unit is based off of this gist from phillips: https://gist.github.com/philips/7e9f7d5990c9e3db65be22943308771d

### CODE REVIEW

**Comment by crawford** [comment]
Please add an Ignition config as well.

File: os/customizing-sshd.md [hunk 1]
```diff
@@ -143,3 +143,73 @@ Enter passphrase for key '/home/user/.ssh/id_rsa':
...
+# systemctl mask sshd.socket
+```
+Finally, execute a daemon-reload, stop the ssh.socket service, and start the sshd.service unit:
+
+```
+# systemctl daemon-reload
+# systemctl stop sshd.socket; systemctl restart sshd.service
+```
+
+This process via cloud-config:
```
Comment (joshix) [inline os/customizing-sshd.md]: The same configuration can be achieved and an actively listening sshd started by providing a cloud-config like:

File: os/customizing-sshd.md [hunk 2]
```diff
@@ -143,3 +143,73 @@ Enter passphrase for key '/home/user/.ssh/id_rsa':
 ### Further reading on systemd units
 
 For more information about configuring CoreOS hosts with `systemd`, see [Getting Started with systemd]({{site.baseurl}}/docs/launching-containers/launching/getting-started-with-systemd/).
+
+### Override socket-activated SSH
+
+Occasionally when systemd gets into a broken state, socket activation doesn't work, which can make a system inaccessible if ssh is the only option. This can be avoided configuring a permanently active SSH daemon that forks for each incoming connection.
+
+We start by creating a unit to override the default sshd unit file at `/etc/systemd/system/sshd.service`:
```
Comment (joshix) [inline os/customizing-sshd.md]:
```
To do this directly on the CoreOS machine, begin by creating overwriting the default sshd unit file at `/etc/systemd/system/sshd.service`: 
```

File: os/customizing-sshd.md [hunk 3]
```diff
@@ -143,3 +143,73 @@ Enter passphrase for key '/home/user/.ssh/id_rsa':
...
+ExecReload=/bin/kill -HUP $MAINPID
+KillMode=process
+Restart=on-failure
+RestartSec=30s
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Next we mask the systemd.socket unit.
```
Comment (joshix) [inline os/customizing-sshd.md]: s/we//
s/./:/

**Comment by crawford** [comment]
LGTM

**Pull request merged**